service/src/main/java/uk/gov/hmcts/reform/fpl/service/validators/ManageLegalRepresentativesValidator.java

```diff
@@ -0,0 +1,102 @@
+        return Streams.mapWithIndex(
+            Optional.ofNullable(legalRepresentatives).orElse(Lists.emptyList()).stream(),
+            (legalRepresentativeElement, idx) ->
+                performBasicValidation(legalRepresentativeElement.getValue(), idx, legalRepresentatives.size())
+        ).flatMap(Collection::stream).collect(Collectors.toList());
+    }
+
+    private List<String> performBasicValidation(LegalRepresentative legalRepresentative, long currentRepresentativeIdx,
+                                                int sizeOfRepresentatives) {
+        List<String> validationErrors = Lists.newArrayList();
+        if (isEmpty(legalRepresentative.getFullName())) {
+            validationErrors.add(validationMessage("Enter a full name",
+                currentRepresentativeIdx, sizeOfRepresentatives));
+        }
+
+        if (isNull(legalRepresentative.getRole())) {
+            validationErrors.add(validationMessage("Select a role",
+                currentRepresentativeIdx, sizeOfRepresentatives));
+        }
+
+        if (isEmpty(legalRepresentative.getOrganisation())) {
+            validationErrors.add(validationMessage("Enter an organisation",
+                currentRepresentativeIdx,
+                sizeOfRepresentatives));
+        }
+
+        if (isEmpty(legalRepresentative.getEmail())) {
+            validationErrors.add(validationMessage("Enter an email address",
+                currentRepresentativeIdx,
+                sizeOfRepresentatives));
+        } else {
+            Optional<String> userId = organisationService.findUserByEmail(legalRepresentative.getEmail());
+            if (userId.isEmpty()) {
+                validationErrors.add(
+                    validationMessageForInvalidEmail(currentRepresentativeIdx, sizeOfRepresentatives));
+            }
+        }
+
+        if (isEmpty(legalRepresentative.getTelephoneNumber())) {
+            validationErrors.add(validationMessage("Enter a phone number",
+                currentRepresentativeIdx,
+                sizeOfRepresentatives)
+            );
+        }
+
+        return validationErrors;
+
+    }
+
+    private String validationMessage(String prefix, long currentRepresentativeIdx, int sizeOfRepresentatives) {
+        return MessageFormat.format("{0} {1}{2}",
+            prefix,
+            VALIDATION_SUFFIX,
+            addNumericIfMultipleElements(currentRepresentativeIdx, sizeOfRepresentatives));
+    }
+
+    private String validationMessageForInvalidEmail(long currentRepresentativeIdx, int sizeOfRepresentatives) {
+        return String.format(
+            "Email address for Legal representative%s is not registered on the system.<br/>They can "
+                + "register at <a href='https://manage-org.platform.hmcts"
+                + ".net/register-org/register'>"
+                + "https://manage-org.platform.hmcts"
+                + ".net/register-org/register</a>",
```

Comment:
I think that this looks nicer when the link lines are merged
```
            "Email address for Legal representative%s is not registered on the system.<br/>They can register at "
                + "<a href='https://manage-org.platform.hmcts.net/register-org/register'>"
                + "https://manage-org.platform.hmcts.net/register-org/register</a>",
```
(intellij doesn't seem to yet support multiline comments 😢)

Reply:
or maybe have the second sentence start on a new line instead as well

Reply:
done